rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc

```diff
@@ -0,0 +1,104 @@
+= Creating a ROSA Hosted Control Plane cluster using the default options
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-sts-creating-a-cluster-quickly
+
+toc::[]
+
+[NOTE]
+====
+If you are looking for a quickstart guide for ROSA, see xref:../rosa_getting_started/rosa-quickstart-guide-ui.adoc#rosa-quickstart-guide-ui[{product-title} quickstart guide].
+====
+
+Create a {hcp} for {product-title} (ROSA) cluster quickly by using the default options and automatic AWS Identity and Access Management (IAM) resource creation. You can deploy your cluster by using the ROSA CLI (`rosa`).
+
+[NOTE]
+====
+All {hcp} for ROSA clusters are only available with AWS Security Token Service (STS) enabled.
+====
+
+The procedures in this document use the `auto` modes in the ROSA CLI (`rosa`) to immediately create the required IAM resources using the current AWS account. The required resources include the account-wide IAM roles and policies, cluster-specific Operator roles and policies, and OpenID Connect (OIDC) identity provider.
+
+Alternatively, you can use `manual` mode, which outputs the `aws` commands needed to create the IAM resources instead of deploying them automatically. For steps to deploy a {hcp} for ROSA cluster by using `manual` mode or with customizations, see xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.adoc#rosa-sts-creating-cluster-using-customizations_rosa-sts-creating-a-cluster-with-customizations[Creating a cluster using customizations].
+
+[id="next-steps-hcp_{context}"]
+.Next steps
+
+* Ensure that you have completed the xref:../rosa_planning/rosa-sts-aws-prereqs.adoc[AWS prerequisites].
+
+include::modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc[leveloffset=+1]
+//include::modules/rosa-sts-understanding-aws-account-association.adoc[leveloffset=+1]
+
+// Commenting this section until OCM functionality is supported
+//[role="_additional-resources"]
+//.Additional resources
+//
+//* For detailed steps to create and link the {cluster-manager} and user IAM roles, see xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc#rosa-sts-associating-your-aws-account_rosa-sts-creating-a-cluster-quickly[Associating your AWS account with your Red Hat organization].
+//
+//include::modules/rosa-sts-creating-a-cluster-quickly-ocm.adoc[leveloffset=+1]
+//include::modules/rosa-sts-associating-your-aws-account.adoc[leveloffset=+2]
+include::modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc[leveloffset=+1]
+[id="rosa-hcp-creating-vpc"]
+== Creating a Virtual Private Cloud for your ROSA HCP clusters
+
+You must have a Virtual Private Cloud (VPC) created to create a {hcp} for ROSA cluster. You can manually create these using the AWS console, or you can use `terraform` to create a VPC by using a template.
+
+include::modules/rosa-hcp-vpc-terraform.adoc[leveloffset=+2]
+
+=== Manual VPC Creation
```

Comment:
@abhatt-rh On this one I'll play around a bit. This section is more of a reference since we aren't explaining how to create a VPC manually but with the aid of AWS. I'll mull it over!